python/plugins/processing/algs/qgis/RectanglesOvalsDiamondsFixed.py

```diff
@@ -58,6 +59,9 @@ def groupId(self):
     def __init__(self):
         super().__init__()
 
+    def flags(self):
```

Comment:
If we register an alias (via QgsApplication::rprocessingRegistry()->addAlgorithmAlias( "qgis:rectanglesovalsdiamondsfixed", "native:rectanglesovalsdiamonds" ) we can just drop this whole file. (I've double checked, and the implementations should give identical results)

Reply:
Done. I'm wondering if we should do the same with "Set raster/vector layer style" algorithms